payjoin-cli/src/app/v2.rs

```diff
@@ -130,25 +153,30 @@ impl AppTrait for App {
         self.db.clear_recv_session()?;
         Ok(())
     }
-}
-
-impl App {
-    fn construct_payjoin_uri(
-        &self,
-        amount_arg: &str,
-        fallback_target: &str,
-        ohttp_keys: payjoin::OhttpKeys,
-    ) -> Result<String> {
-        let pj_receiver_address = self.bitcoind()?.get_new_address(None, None)?.assume_checked();
-        let amount = Amount::from_sat(amount_arg.parse()?);
-        let pj_part = payjoin::Url::parse(fallback_target)
-            .map_err(|e| anyhow!("Failed to parse Payjoin subdirectory target: {}", e))?;
 
-        let pj_uri = PjUriBuilder::new(pj_receiver_address, pj_part, Some(ohttp_keys))
-            .amount(amount)
-            .build();
+    pub async fn resume_payjoins(&self) -> Result<()> {
+        let self_clone = self.clone();
+        let mut tasks = Vec::new();
 
-        Ok(pj_uri.to_string())
+        let recv_sessions = self.db.get_recv_sessions()?;
+        for recv_session in recv_sessions {
+            let self_clone = self_clone.clone();
+            tasks.push(tokio::task::spawn(async move {
+                self_clone.spawn_payjoin_receiver(recv_session, None).await
+            }));
+        }
+        let send_sessions = self.db.get_send_sessions()?;
+        for send_session in send_sessions {
+            let self_clone = self_clone.clone();
+            tasks.push(tokio::task::spawn(async move {
+                self_clone.spawn_payjoin_sender(send_session).await
+            }));
+        }
+        for task in tasks {
+            let _ = task.await?;
+        }
+        println!("All resumed sessions completed.");
```

Comment:
It would be helpful to print a "nothing to do" message if there are no sessions to resume.